src/test/smart-components/workflow/workflows.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import thunk from 'redux-thunk';
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store' ;
+import { shallowToJson } from 'enzyme-to-json';
+import promiseMiddleware from 'redux-promise-middleware';
+
+import Workflows from '../../../smart-components/workflow/workflows';
+import { workflowsInitialState } from '../../../redux/reducers/workflow-reducer';
+import { notificationsMiddleware } from '@redhat-cloud-services/frontend-components-notifications';
+import { groupsInitialState } from '../../../redux/reducers/group-reducer';
+
+describe('<Workflows />', () => {
+
+  let initialProps;
+  const middlewares = [ thunk, promiseMiddleware(), notificationsMiddleware() ];
+  let mockStore;
+  let initialState;
+
+  beforeEach(() => {
+    initialProps = {};
+    mockStore = configureStore(middlewares);
+    initialState = { workflowReducer: { ...workflowsInitialState, isLoading: false }, groupReducer: { ...groupsInitialState }};
+  });
+
+  afterEach(() => {
```

Comment:
Fetch mock is not used anywhere in this test file you can delete `afterEach` method.

Reply:
Done